src/dex-ui/pages/Governance/Governance.tsx

```diff
@@ -72,88 +72,91 @@ export const Governance = (): ReactElement => {
   const handleClickNewProposalButton = () => navigate("/governance/select-proposal-type");
 
   return (
-    <Grid templateColumns="repeat(1, 1fr)" gap={8} width="100%">
```

Comment:
No major changes to this file - I've wrapped the header and content of this page in the new `Page` layout component for a more consistent experience.